test/policy_test.exs

```diff
@@ -19,10 +19,7 @@ defmodule Wanda.PolicyTest do
     agent_id = UUID.uuid4()
     env = build(:execution_requested_env)
 
-    expected_env =
-      Enum.reduce(env, %{}, fn {key, %{kind: {_, value}}}, acc ->
-        Map.put(acc, key, value)
-      end)
+    expected_env = Map.new(env, fn {key, %{kind: {_, value}}} -> {key, value} end)
```

Comment:
we are repeating the implementation in the test, which makes the test of little value.

Reply:
Ah yeah! Glad you spotted! I overlooked that. My bad :sweat_smile: